scripts/test_env.sh

```diff
@@ -141,7 +146,7 @@ case $1 in
             *)
                 clean_up_peers
         esac
-        rm "$TEST" -r -f
+        rm -rf "$TEST"
```

Comment:
Let's not get into this argument

Reply:
hmm, my mac just don't delete anything if i run `rm "$TEST" -r -f` because it's interpreting `-r -f` as files